ssh/tailssh/incubator.go

```diff
@@ -691,6 +691,9 @@ func (ss *sshSession) startWithStdPipes() (err error) {
 func loginShell(u *user.User) string {
 	switch runtime.GOOS {
 	case "linux":
+		if distro.Get() == distro.Gokrazy {
+			return "/tmp/serial-busybox/ash"
```

Comment:
Likewise, I’m a little concerned about hard-coding the serial-busybox path here.

(Maybe we’ll eventually follow the suggestion to use u-root instead of busybox, for example.)

Maybe we should create /bin/sh on gokrazy when gokrazy finds ash? So that we only need to hard-code it in one place, not in downstream applications like tailscale?